docs/zh_cn/build/windows.md

```diff
@@ -0,0 +1,336 @@
+    - [安装构建和编译工具链](#安装构建和编译工具链)
+    - [安装依赖包](#安装依赖包)
+      - [安装 MMDeploy Converter 依赖](#安装-mmdeploy-converter-依赖)
+      - [安装 MMDeploy SDK 依赖](#安装-mmdeploy-sdk-依赖)
+      - [安装推理引擎](#安装推理引擎)
+    - [编译 MMDeploy](#编译-mmdeploy)
+      - [编译安装 Model Converter](#编译安装-model-converter)
+        - [编译自定义算子](#编译自定义算子)
+        - [安装 Model Converter](#安装-model-converter)
+      - [编译 SDK](#编译-sdk)
+        - [编译选项说明](#编译选项说明)
+        - [编译样例](#编译样例)
+        - [编译 SDK Demo](#编译-sdk-demo)
+    - [注意事项](#注意事项)
+
+---
+# Windows 下构建方式
+
+目前，MMDeploy 在 Windows 平台下仅提供源码编译安装方式。未来会提供预编译包方式。
+
+## 源码安装
+下述安装方式，均是在 **Windows 10** 下进行
+### 安装构建和编译工具链
+1. 下载并安装 [Visual Studio 2019](https://visualstudio.microsoft.com) 。安装时请勾选 "使用C++的桌面开发, "Windows 10 SDK <br>
+2. 把 cmake 路径加入到环境变量 PATH 中, "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin" <br>
+3. 如果系统中配置了 NVIDIA 显卡，根据官网教程，下载并安装 cuda toolkit。<br>
+### 安装依赖包
+
+#### 安装 MMDeploy Converter 依赖
+<table>
+<thead>
+  <tr>
+    <th>名称 </th>
+    <th>安装方法 </th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>conda </td>
+    <td>强烈建议安装conda，或者miniconda。比如， <br>https://repo.anaconda.com/miniconda/Miniconda3-py37_4.11.0-Windows-x86_64.exe <br>安装完毕后，打开系统开始菜单，输入prompt，选择并打开 anaconda powershell prompt。 <br><b>下文中的安装命令均是在 anaconda powershell 中测试验证的。</b> </td>
+  </tr>
+  <tr>
+    <td>pytorch <br>(>=1.8.0) </td>
+    <td>
+    参考<a href="https://pytorch.org/get-started/locally/">pytorch官网</a>，根据系统环境, 选择合适的预编译包进行安装。比如, <br>
+    <pre><code>
+    pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 torchaudio==0.8.1 -f https://download.pytorch.org/whl/torch_stable.html
+    </code></pre>
+    </td>
+  </tr>
+  <tr>
+    <td>mmcv-full </td>
+    <td>参考<a href="https://github.com/open-mmlab/mmcv">mmcv官网</a>，根据系统环境，选择预编译包进行安装。比如，<br>
+    <pre><code>
+    $env:cu_version="cu111"
+    $env:torch_version="torch1.8.0"
+    pip install mmcv-full==1.4.0 -f https://download.openmmlab.com/mmcv/dist/$env:cu_version/$env:torch_version/index.html
+    </code></pre>
+    </td>
+  </tr>
+</tbody>
+</table>
+
+
+#### 安装 MMDeploy SDK 依赖
+<table>
+<thead>
+  <tr>
+    <th>名称 </th>
+    <th>安装方法 </th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>spdlog </td>
+    <td>spdlog是一个精巧的日志管理库。请参考如下命令安装： <br>
+    1. 下载 https://github.com/gabime/spdlog/archive/refs/tags/v1.9.2.zip <br>
+    2. 解压后，进入到文件夹 spdlog-v1.9.2 <br>
+    3. 执行编译安装命令 <br>
+    <pre><code>
+    mkdir build
+    cd build
+    cmake .. -G "Visual Studio 16 2019" -A x64 -T v142 -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_INSTALL_PREFIX=install -DCMAKE_BUILD_TYPE=Release
+    cmake --build . --target install -j --config Release
+    </code></pre>
+   </td>
+  </tr>
+  <tr>
+    <td>OpenCV </td>
+    <td>
+    1. 下载并安装 OpenCV 在 windows 下的预编译包: https://github.com/opencv/opencv/releases/download/4.5.5/opencv-4.5.5-vc14_vc15.exe <br>
+    2. 把 OpenCV 库的路径加入到环境变量 PATH 中</td>
+
+  </tr>
+  <tr>
+    <td>pplcv </td>
```

Comment:
"pplcv" requires to clone a lot of git dependencies, which may fail in a poor network environment (specially, pplcommon is too large to download). We may ask the user to download the repos manually. I resolved this issue by changing the links in 'ppl.cv/ cmake/deps.cmake' from http link to git link.

Reply:
Only using ppl.cv's git repo can generate VERSION macro in `pplcv-config.cmake`. The VERSION macro is necessary for MMDeploy to do cuda transform ops compatibility. I think it's better for ppl.cv to change its VERSION generation method.